submissions/youhymuk/html-css-popup/styles/styles.css

```diff
@@ -0,0 +1,199 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+a:hover, a:focus {
+  outline: none;
+  background-color: rgba(64, 169, 255, 0.4);
+}
+
+img {
+  max-width: 100%;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.toolbar {
+  height: 50px;
+  min-width: 32px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 0 2%;
+}
+.toolbar-item:not(:last-child) {
+  margin-right: 1rem;
+}
+
+.popup-wrapper {
+  position: relative;
+}
+
+.popup-btn {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+}
+.popup-btn:focus ~ .popup-icon {
+  background-color: rgba(64, 169, 255, 0.4);
+}
+.popup-btn:checked ~ .popup {
+  display: block;
+}
+
+.popup-icon {
+  display: block;
+  width: 24px;
+  height: 24px;
+  border-radius: 50%;
+  cursor: pointer;
+  background: url(../img/popup-button.png) center/cover no-repeat;
+}
+.popup-icon:hover {
+  background-color: rgba(64, 169, 255, 0.4);
+}
+
+.bell {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+}
+.bell img {
+  width: 18px;
+}
+
+.user-acc {
+  display: block;
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  border: 1px solid #c5c5c5;
+}
+
+.user-pic {
+  max-width: 100%;
+}
+
+.popup {
+  position: absolute;
+  top: 50px;
+  right: -90px;
+  display: none;
+}
```

Comment:
It is better to write everywhere in the same style and add a blank line between the blocks.

Reply:
Oh, I know it and this is exactly what I did in my scss file, but the compiler removed it(
I'll keep this in mind in the future, thanks!